lib/index.js

```diff
@@ -192,27 +153,13 @@ export default {
           return [];
         }
 
-        // Construct the list of linters to be passed to the CLI by looking at
-        // the linters made available for our language grammar by the LinterMap,
-        // and whether or not the engines are enabled in a user's config file.
-        if (Object.prototype.hasOwnProperty.call(linterMap, grammarName) === true) {
-          // for (const linter of linterMap[grammarName]) {
-          linterMap[grammarName].forEach((linter) => {
-            linterNames.add(linter);
-          });
-        }
-        const configEnabledEngines = getEnabledEngines(configurationFilePath);
-        const linterEnabledEngines = Array.from(linterNames).filter(
-          linterName => configEnabledEngines.has(linterName));
-
         // Construct the command line invocation which runs the Code Climate CLI
         const relpath = atom.project.relativizePath(filePath).pop();
-        const execArgs = ['analyze', '-f', 'json'].concat(
-          getEngineArgs(linterEnabledEngines),
-          [relpath, '</dev/null']);
+        const execArgs = ['analyze', '-f', 'json', relpath];
         const execOpts = {
           cwd: dirname(configurationFilePath),
           uniqueKey: `linter-codeclimate::${relpath}`,
+          env: process.env,
```

Comment:
It's interesting that you mention this.

For me the the specs were failing the way they fail right now on CI without this line. I hadn't committed it at first thinking that it's just my weird setup (Docker on MacOS runs on non-default ip/port combination for some reason). When I added this line my local setup started picking up `DOCKER_HOST` var and all was well.

That's why when I saw the failure on CI I pushed this line as well. But it appears to be something else this time.

I'd appreciate any hint anyone can give me regarding the CI failure.